changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v0.7.2 | t.b.d.
+* Change `h5py` requirement that the version should be smaller than 3 to bigger or equal to three.
```

Comment:
* Add a blank like after Markdown (section) titles
* The changelog should really mention something about adding Python 3.9 support on Windows (I assume that things worked on Linux before?)
* The sentence is a bit confusing. Maybe simply: `Bumped h5py requirement to >= 3.0`

Reply:
How about `* Support Python 3.9 on Windows (this required bumping the `h5py` requirement to >= 3.0).`?